### PR TITLE
Fix integration config loading, introduced NoError in tests

### PR DESCRIPTION
**Pull request by lzap**
Fixes configuration of tests, add missing ENV variable and adds NoError in all DAO tests.

Three commits, see commit messages for more details.

ACTION REQUIRED: This patch also fixes configuration loading, it now properly finds the path to the config file. Please move your `cmd/dao_test/local.yaml` to `configs/local.integration.yaml` to get it loaded properly.

### CODE REVIEW

**Review by avitova — APPROVED** [review]
LGTM

File: internal/config/5_normal_config.go [hunk 1]
```diff
@@ -4,38 +4,14 @@
...
-		// load from working directory for integration tests
-		parser.Viper.AddConfigPath(".")
-		parser.Viper.SetConfigName("local")
+	parser.Viper.SetDefault("featureFlags.environment", "development")
 
-		err := parser.Viper.ReadInConfig()
-		if err != nil {
-			fmt.Println("Could not read configs/local.yaml, continuing with defaults")
-		}
-	}
```
Comment (lzap) [inline internal/config/5_normal_config.go:40]: FYI this was the reason why it did not work - this file is now excluded when integration build tag is set. So I moved the logic into the Initialize() function where it probably belongs.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @lzap ! :)

File: internal/config/5_normal_config.go (hunk 1 above)
Comment (ezr-ondrej) [reply internal/config/5_normal_config.go:40]: I'd agree, we (read I) went overboard with the splitting the config into files.

**Pull request merged**